### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by veerlevijverberg**

### CODE REVIEW

**Review by MagdaMatetovici — COMMENTED** [review]
Your Vignette is already almost finished, it looks good. In your Shiny code, I suggest adding more detailed comments. For your videos, you should have a "www" folder in your R folder which contains the video materials - right now you use the path from your computer and this will not work when someone else runs it on another computer. You should use the path to your www folder instead.

**Comment by L-Groot** [comment]
Your project looks almost complete! I see testing functions and you have a very detailed vignette.
I agree with Magda's suggestions and I have two small suggestions for the vignette:
- I think it could be cool to move the conclusion (currently at the end of the vignette) to the top. Similar to an abstract, it would be a nice pitch that gives the reader an idea of your app at the first glance.
- You could remove the knitr calls that generate the screenshots in the vignette.

File: Vignette.pdf
Comment (vincentott) [inline Vignette.pdf:1]: I agree with Laura, that the conclusion could also be moved to the top of the vignette.
Otherwise the vignette is well structured.

**Review by vincentott — APPROVED** [review]
It seems that the main branch as well as the week3 and wek4 branch do not have the typical R package structure. For example, there is no folder "R" containing R scripts and the vignette is not in a "vignettes" folder. I suggest that you re-structure your project a bit so that it is more accessible to programmers who are not familiar with your project but are used to the typical R package structure.

**Pull request merged**